lib/matplotlib/typing.py

```diff
@@ -69,7 +70,16 @@
 )
 """See :doc:`/gallery/lines_bars_and_markers/markevery_demo`."""
 
-MarkerType: TypeAlias = str | path.Path | MarkerStyle
+MarkerType: TypeAlias = (
+    path.Path | MarkerStyle | str |  # str required for "$...$" marker
+    Literal[
```

Comment:
Since we have to keep str anyway, what is the advantage of spelling out the literal string values?

Reply:
Sorry for the silence.
Just wanted to check: would you prefer that I remove the string literals? I'll adjust the code accordingly.

Reply:
Ok, let's keep them.